### PR TITLE
Modal Styling

### LINKED ISSUE #65: Modal Styling
**Is your feature request related to a problem? Please describe.**
Change background color to reflect pokemon type of selected pokemon

**Describe the solution you'd like**
Use pokemon type to chang the background color of the modal according to the pokemon you are viewing

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Modal background color represents pokemon type
2.  Function to capitalize first letter of pokemon type
3. Basic styles to modal background

## Purpose
For the pokemon type color to change according to type, and improve general styling of modal.

Closes #65

### CODE REVIEW

File: src/components/PokeList.js [hunk 1]
```diff
@@ -143,18 +135,28 @@ const PokeList = ({
...
       const typeName = obj["type"]["name"];
+      const capitalizedTypeName = capitalizeType(typeName);
+      console.log(`capitalized Type Name `, capitalizedTypeName);
       return (
         <span
           className="type-tag"
-          style={{ backgroundColor: getTypeColors[typeName] }}
+          style={{
+            backgroundColor: getTypeColors[typeName],
+            // textTransform: "uppercase",
```
Comment (dannysanchez559) [inline src/components/PokeList.js:156]: take care of this lingering comment

**Review by dannysanchez559 — CHANGES_REQUESTED** [review]
"loving the logic!"

**Review by dannysanchez559 — APPROVED** [review]
LoOKs GoOd tO mEee!

File: src/components/Modal.js [hunk 2]
```diff
@@ -38,33 +38,38 @@ export const Modal = ({
...
           <div className="nameContainer">
-            <h1 className="modalPokeName">{name}</h1>
-            <button onClick={playPokemonCry} ref={speakerButtonRef}>
-              <img src={crySpeaker} ref={speakerRef} alt="speaker"/>
-            </button>
-            <span>{typeTags}</span>
+            <div className="nameSpeakerContainer">
+              <h1 className="modalPokeName">{name}</h1>
+              <button onClick={playPokemonCry} ref={speakerButtonRef}>
+                <img src={crySpeaker} ref={speakerRef} />
```
Comment (BlackCubes) [inline src/components/Modal.js:69]: You had an `alt` attribute before, but you removed it? Is that kewl for yo' team?

File: src/components/PokeList.js [hunk 3]
```diff
@@ -143,18 +135,27 @@ const PokeList = ({
     // eslint-disable-next-line
   }, [sortType, filterType, reverse, selectTypeOption, selectAbilityOption]);
 
+  const capitalizeType = (typeString) => {
+    const capitalize = typeString[0].toUpperCase() + typeString.slice(1);
+    return capitalize;
+  };
+
```
Comment (BlackCubes) [inline src/components/PokeList.js:142]: Nice function!

File: src/components/PokeList.js [hunk 4]
```diff
@@ -143,18 +135,27 @@ const PokeList = ({
...
   // generate tags, check for tags and types
   let typeTags, sprite;
   if (types && sprites) {
     sprite = sprites["other"]["official-artwork"]["front_default"];
     typeTags = types.map((obj, i) => {
       const typeName = obj["type"]["name"];
+      const capitalizedTypeName = capitalizeType(typeName);
+      console.log(`capitalized Type Name `, capitalizedTypeName);
```
Comment (BlackCubes) [inline src/components/PokeList.js:150]: Muh man! Get rid of dem `console.log`!

File: src/components/PokeList.js [hunk 5]
```diff
@@ -143,18 +135,27 @@ const PokeList = ({
...
       return (
         <span
           className="type-tag"
-          style={{ backgroundColor: getTypeColors[typeName] }}
+          style={{
+            backgroundColor: getTypeColors[typeName],
+          }}
           key={i}>
-          {obj["type"]["name"]}{" "}
+          {capitalizedTypeName}
```
Comment (BlackCubes) [inline src/components/PokeList.js:158]: This looks better than before

**Review by BlackCubes — APPROVED** [review]
Everything looks good!

File: src/components/Modal.js (hunk 2 above)
Comment (Gjhnsn) [reply src/components/Modal.js:69]: ooooh good catch! Thanks!

**Review by BlackCubes — COMMENTED** [review]
Good changes!

**Pull request merged**